### PR TITLE
Created tests for Tag and Topic

### PR DESCRIPTION
**Pull request by seongchanlee**

### CODE REVIEW

File: dispatch/tests/test_tags.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from dispatch.tests.cases import DispatchAPITestCase
+
```
Comment (psiemens) [inline dispatch/tests/test_tags.py]: Remove this line break

File: dispatch/tests/test_tags.py [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+from dispatch.apps.content.models import Tag
+
+class tagsTests(DispatchAPITestCase):
```
Comment (psiemens) [inline dispatch/tests/test_tags.py]: Capitalize class names

File: dispatch/tests/test_tags.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    def _create_tag(self):
+        """
+        Create a dummy tag instance
```
Comment (psiemens) [inline dispatch/tests/test_tags.py]: Format method docstrings on single line:

```py
def _create_tag(self):
  """Create a dummy tag instance"""
```

File: dispatch/tests/test_tags.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+        url = reverse('api-tags-list')
+
+        return self.client.post(url, data, format='json')
+
+    def test_create_tag_unauthorized(self):
+
+        """
+		Create tag should fail with unauthenticated request
+        """
```
Comment (psiemens) [inline dispatch/tests/test_tags.py]: Fix formatting

File: dispatch/tests/test_tags.py [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+    def test_create_incomplete_tag(self):
+        """
+        Create tag should fail with missing required fields
+        """
+
+        url = reverse('api-tags-list')
+
+        # tag data is missing name
+        data = {
```
Comment (psiemens) [inline dispatch/tests/test_tags.py]: Change to:
```py
data = {}
```

File: dispatch/tests/test_topics.py [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+        url = reverse('api-topics-list')
+
+        return self.client.post(url, data, format='json')
+
+    def test_create_topic_unauthorized(self):
+
+        """
+		Create topic should fail with unauthenticated request
+		"""
```
Comment (psiemens) [inline dispatch/tests/test_topics.py]: Fix formatting

File: dispatch/tests/test_topics.py [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+
+    def test_create_incomplete_topic(self):
+        """
+        Create topic should fail with missing required fields
+        """
+
+        url = reverse('api-topics-list')
+
+        # topic data is missing name
+        data = {
```
Comment (psiemens) [inline dispatch/tests/test_topics.py]: Format on single line

**Review by psiemens — CHANGES_REQUESTED** [review]
Just a few minor style fixes 😄

**Comment by seongchanlee** [comment]
Woah what is this massive coverage increase

**Review by psiemens — APPROVED** [review]

**Pull request merged**